Documentation/Settings.cfg

```diff
@@ -39,8 +39,8 @@ copyright   = by {extension.extensionKey}
 # ...  (recommended) to get the "Edit me on Github Button"
 # .................................................................................
 
-github_branch        = master
-github_repository    = TYPO3-Documentation/TYPO3CMS-Example-ExtensionManual
+#github_branch        = master
```

Comment:
Should we add a hint to comment this out? As this is an example for every beginner to start with, we should explain the necessary steps and don't assume any prior knowledge. What do you think?

Reply:
What do you have in mind? A description what "edit me on github" does?

Reply:
Just a hint that `#` is a comment, and that authors should remove the comment and adjust the repository.

Reply:
Hm... I honestly think this is something we can expect from extension authors to recognize. Simply by scrolling over the file you immediately spot that free text is always beginning with a `#`.